lib/diagnostic.rb

```diff
@@ -124,16 +148,23 @@ def punch
     @moves << 'punch'
     @damage += 5
     @damage *= multiplier
+    self
   end
 
   def move(direction)
     @moves << "move #{direction}"
+    self
   end
 
   def kick
     @moves << 'kick'
     @damage += 10
     @damage *= multiplier
+    self
+  end
+
+  def get_possible_moves
+    "#{@kick}, #{@move}, #{@punch}"
```

Comment:
just `"kick, move, punch"` because there are no kick move and punch instance variables